plugins/ruby-foreman-tasks/debian/gem.list

```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-5.2.0.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-5.3.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.4.0.gem"
```

Comment:
I think this can be dropped now?

Reply:
salt core depends on it. Can I drop it before salt core goes away?

Reply:
to hell with salt core, it's not installable right now anyways (it depends on rex-core, which is gone)